### PR TITLE
ADM-589 Add time duration display in UI and csv file name

### PR DESCRIPTION
**Pull request by 7758521yyy**
## Summary

- Add the collection duration display in both `metrics` and `report` page.
- Add the collection duration info in the exported csv file name.

## Before

image

## After

image

## Note

_Null_

### CODE REVIEW

File: frontend/src/components/Common/CollectionDuration/style.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  letter-spacing: 0.1em;
+  padding: 0.25rem 0;
+  font-weight: bold;
+  margin-bottom: 0.125rem;
+`
+
+export const DateText = styled('div')`
+  font-size: 1.25rem;
+  letter-spacing: 0.1em;
+  font-family: monospace;
```
Comment (guzhongren) [inline frontend/src/components/Common/CollectionDuration/style.tsx:73]: no `font-family`

File: frontend/src/components/Common/CollectionDuration/style.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+`
+
+export const MonthYearText = styled('div')`
+  font-size: 0.625rem;
+  margin-bottom: 0.125rem;
+  font-family: monospace;
```
Comment (guzhongren) [inline frontend/src/components/Common/CollectionDuration/style.tsx:79]: no `font-family`

File: frontend/src/components/Common/CollectionDuration/index.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+type DisplayedDateInfo = {
+  year: string
+  month: string
+  day: string
+}
+
+const CollectionDuration = ({ startDate, endDate }: Props) => {
+  const startDateInfo: DisplayedDateInfo = {
```
[the remark this answers is not in the document]
Comment (7758521yyy) [reply frontend/src/components/Common/CollectionDuration/index.tsx:26]: > Just use 1 `dayjs(startDate)` not create it again in the rest part

you mean like this?
```
  const toBeFormattedStartDate = dayjs(startDate)
  const startDateInfo: DisplayedDateInfo = {
    year: toBeFormattedStartDate.format('YY'),
    month: toBeFormattedStartDate.format('MMM'),
    day: toBeFormattedStartDate.format('DD'),
  }
```

File: frontend/src/components/Common/CollectionDuration/style.tsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+export const CollectionDateContainer = styled('div')`
+  display: flex;
+  align-items: flex-end;
+  margin: 2rem auto 0;
+`
+styled('strong')`
+  margin-left: 0.25rem;
+  font-size: 0.875rem;
+`
```
[the remark this answers is not in the document]
Comment (7758521yyy) [reply frontend/src/components/Common/CollectionDuration/style.tsx:12]: > what is it?

Sorry, forget to delete

**Review by guzhongren — APPROVED** [review]
A1 comment

**Pull request merged**